File: NearbyPlaces/UIImageView+TintColor.m
//
//  UIImageView+TintColor.m
//  NearbyPlaces
//
//  Created by Adrian Kubała on 13.02.2017.
//  Copyright © 2017 Adrian Kubała. All rights reserved.
//

#import "UIImageView+TintColor.h"

@implementation UIImageView (TintColor)

-(void)changeTintColor:(UIColor *)color {
  self.image = [self.image imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate];
  self.tintColor = color;
}

@end

File: Places/UIImageView+tintColor.swift
//
//  UIImageView+tintColor.swift
//  Places
//
//  Created by Adrian on 28.09.2016.
//  Copyright © 2016 Adrian Kubała. All rights reserved.
//

import UIKit

extension UIImageView {
  func changeTintColor(_ color: UIColor) {
    image = image?.withRenderingMode(.alwaysTemplate)
    tintColor = color
  }
}
